### ISSUE TITLE
ESP32-S3: Get USB Host (OTG) working

### ISSUE THREAD

**Issue by K7MDL2** [K7MDL2/IC-705-BLE-Serial-Example#21]
Some controller models have an OTG capable USB port  The M5StampC3U has 2 USB ports, as do some dev boards.  The ESP32-S3 models have an OTG type C USB jack connected to the CPU internally.  The 2nd USB is via a pin header to an onboard USB serial converter chip (CP2104).

At least 1, maybe both support USB host mode.  The CoreS3 USB port is OTG (dual mode).

Task is to get USB host working on one of the ports and connect to a radio.  Then setup BT as a server for a connection from PC.

Stretch goal is to use BT Mesh if possible (likely not at the same time as BT or BLE on the radio) to connect a radio, a PC and a distant decoder breakout, say one located on the back of a dish.  No wires!

This is all fine for control but does nothing for audio connection.  Most r of the modern Icom radios support network connection (LAN or WiFi) which seems to work in parallel.   It is possible to get the BT audio working, nothing I have ever tried before but there are examples.

**Comment by K7MDL2** [comment]
I have USB OTG host working on the IC-905 band decoder. It has a PCB and supports the IC-705 and IC-9700.  The 9700 required VCP drivers.  The 705 works great using BT so USB host on this version offers little ROI.  However, it is useful for the 905 and 9700.  Since the PCB ESP32-S3 version has dual USB ports, one being OTG, it is better suited for supporting the 9700 and 905, both which lack BT.   That version currently does not have transverter support feature that this version has.  So either way code must be ported to one or the other.  

For now, I have no plans to add USB host support to this version, instead opting to add Xvtr and CI-V pass-through features to the dual USB port version at [https://github.com/K7MDL2/IC-905_Band_Decoder_on_ESP32-S3](url).
